src/engraving/draw/pen.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_DRAW_PEN_H
+#define MU_DRAW_PEN_H
+
+#include <QColor>
+#include "drawtypes.h"
+#include "global/log.h"
+
+namespace mu::draw {
+
+class Pen {
+public:
+    
+    Pen() { setDefaults(); }
+    
+    Pen(PenStyle style) {
+        setDefaults();
+        _style = style;
+    }
+    
+    Pen(const QColor &color) {
+        setDefaults();
+        _brush = color;
+    }
+    
+    Pen(const QBrush &brush, double width, PenStyle s = SolidLine,
```

Comment:
I believe that `QBrush` is not needed, this constructor can be replaced with `Pen(const QColor &color, double width,...)`